Gemfile

```diff
@@ -135,6 +135,7 @@ group :development do
   gem 'memory_profiler'
   gem 'rubocop', '~> 1.30', require: false
   gem 'rubocop-rails', '~> 2.15', require: false
+  gem 'rubocop-rspec', '~> 2.11', require: false
```

Comment:
`2.11.1` seems to be the latest version we can use due to our lock to `1.30.x` of `rubocop`.